testcases/two_zone_apartment_hydronic/models/TwoZoneApartmentHydronic/TestCases/ApartmentModelQHTyp.mo

```diff
@@ -0,0 +1,1068 @@
+</li>
+<li>
+<code>weatherStation_reaWeaLat_y</code> [rad] [min=None, max=None]: Latitude of the location
+</li>
+<li>
+<code>weatherStation_reaWeaLon_y</code> [rad] [min=None, max=None]: Longitude of the location
+</li>
+<li>
+<code>weatherStation_reaWeaNOpa_y</code> [1] [min=None, max=None]: Opaque sky cover measurement
+</li>
+<li>
+<code>weatherStation_reaWeaNTot_y</code> [1] [min=None, max=None]: Sky cover measurement
+</li>
+<li>
+<code>weatherStation_reaWeaPAtm_y</code> [Pa] [min=None, max=None]: Atmospheric pressure measurement
+</li>
+<li>
+<code>weatherStation_reaWeaRelHum_y</code> [1] [min=None, max=None]: Outside relative humidity measurement
+</li>
+<li>
+<code>weatherStation_reaWeaSolAlt_y</code> [rad] [min=None, max=None]: Solar altitude angle measurement
+</li>
+<li>
+<code>weatherStation_reaWeaSolDec_y</code> [rad] [min=None, max=None]: Solar declination angle measurement
+</li>
+<li>
+<code>weatherStation_reaWeaSolHouAng_y</code> [rad] [min=None, max=None]: Solar hour angle measurement
+</li>
+<li>
+<code>weatherStation_reaWeaSolTim_y</code> [s] [min=None, max=None]: Solar time
+</li>
+<li>
+<code>weatherStation_reaWeaSolZen_y</code> [rad] [min=None, max=None]: Solar zenith angle measurement
+</li>
+<li>
+<code>weatherStation_reaWeaTBlaSky_y</code> [K] [min=None, max=None]: Black-body sky temperature measurement
+</li>
+<li>
+<code>weatherStation_reaWeaTDewPoi_y</code> [K] [min=None, max=None]: Dew point temperature measurement
+</li>
+<li>
+<code>weatherStation_reaWeaTDryBul_y</code> [K] [min=None, max=None]: Outside drybulb temperature measurement
+</li>
+<li>
+<code>weatherStation_reaWeaTWetBul_y</code> [K] [min=None, max=None]: Wet bulb temperature measurement
+</li>
+<li>
+<code>weatherStation_reaWeaWinDir_y</code> [rad] [min=None, max=None]: Wind direction measurement
+</li>
+<li>
+<code>weatherStation_reaWeaWinSpe_y</code> [m/s] [min=None, max=None]: Wind speed measurement
+</li>
+</ul>
+<h3>Additional System Design</h3>
+<h4>Lighting</h4>
+<p>
+Artificial lighting is provided by LED lights in the thermal zones.
+</p>
+<h4>Shading</h4>
+<p>
+There are no shades on the building.
+</p>
+<h4>Onsite Generation and Storage</h4>
+<p>
+There is no energy generation or storage on the site.
+</p>
+<h3>Model Implementation Details</h3>
+<h4>Moist vs. dry air</h4>
+<p>
+A moist air model is used, but condensation is not modeled in the HVAC
+and humidity is not monitored.
+
+</p>
+<h4>Pressure-flow models</h4>
+<p>
+The pump is an ideal constant head pump and provided each floor heating circuit with exactly 620 [l/h] if the circuit valve is open.
+</p>
+<h4>Infiltration models</h4>
+<p>
+A constant infiltration flowrate is assumed to be 0.5 ACH.
+</p>
+<h4>Other assumptions</h4>
+<p>
+No further assumptions are needed.
+</p>
+<p>
+CO2 generation is modelled but no generation is considered. So the CO2 concentration will be in equilibrium with the external environment CO2.
```

Comment:
The section on internal loads and schedules says CO2 generation is modeled.  Please clarify this section or the other.  Also, "CO2 generation is modelled but no generation is considered" is not clear.

Reply:
Removed this part since it's outdated with respect to the description in the internal loads